src/main/java/org/sonarsource/auth/github/GitHubSettings.java

```diff
@@ -120,6 +121,12 @@ public String loginStrategy(){
         .defaultValue(LOGIN_STRATEGY_DEFAULT_VALUE)
         .options(LOGIN_STRATEGY_UNIQUE, LOGIN_STRATEGY_PROVIDER_ID)
         .index(5)
+      PropertyDefinition.builder(ORGANIZATION)
```

Comment:
Don't you think that plugin should allow to configure multiple organizations, but not only one ?